CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+
+# TODO: Start to fill out a full guide here
+
+## Upgrade Guidelines
+
+The following changes are considered safe for patching:
+
+1. Adding new types (with corresponding additions to `types.json`
+2. Appending to enum types
+3. Adding new storage items
+4. Adding or changing event types
+5. Adding extrinsics
+6. Adding new RPCs
```

Comment:
Should we caveat about RPCs runtime vs native and requiring hard-fork or not to use? Also I guess we'll need to figure out how OCWs fit here